src/devices/Mcp23xxx/BusAdapter.cs

```diff
@@ -7,7 +7,7 @@
 
 namespace Iot.Device.Mcp23xxx
 {
-    public abstract partial class Mcp23xxx : IGpioController
+    public abstract partial class Mcp23xxx
```

Comment:
Do those need some property/base class to make it possible to use them as GpioController?

Reply:
nvm, it's there